tests/run-tests.py

```diff
@@ -165,6 +165,8 @@ def send_get(what):
             cmdlist = [MICROPYTHON, "-X", "emit=" + args.emit]
             if args.heapsize is not None:
                 cmdlist.extend(["-X", "heapsize=" + args.heapsize])
+            if sys.platform == "darwin":
+                cmdlist.extend(["-X", "realtime"])
```

Comment:
Please put this change, and the change to `tools/ci.sh`, in a separate commit.

Reply:
done